diff --git a/cmake/FindProtobuf.cmake b/cmake/FindProtobuf.cmake
index 35625b1e2e..b962448206 100644
--- a/cmake/FindProtobuf.cmake
+++ b/cmake/FindProtobuf.cmake
@@ -22,7 +22,7 @@ Protobuf_FOUND          - True if Protobuf found.
 #]=======================================================================]
 find_package(PkgConfig REQUIRED)
 
-pkg_check_modules(PROTOBUF REQUIRED protobuf>=3.12 IMPORTED_TARGET GLOBAL)
+pkg_check_modules(PROTOBUF REQUIRED protobuf>=3.11 IMPORTED_TARGET GLOBAL)
 add_library(protobuf::libprotobuf ALIAS PkgConfig::PROTOBUF)
 set_target_properties(PkgConfig::PROTOBUF PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES "${PROTOBUF_INCLUDEDIR}")
